client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -78,34 +85,47 @@ const useInvestigationForm = (parameters: useInvestigationFormIncome): useInvest
         timeout(1900).then(() => history.push(landingPageRoute));
     };
 
+    const handleSwitchTab = () => {
+        switch(currentTab.name) {
+            case(TabNames.PERSONAL_INFO): {
+                savePersonalInfoData()
+            }
+            case(TabNames.CLINICAL_DETAILS): {
+                saveClinicalDetails();
+            }
+        }
+
+        setCurrentTab(tabs[currentTab.id + 1]);
```

Comment:
Do you want to allow switching tabs when the save failed?

Reply:
YOYOYOYO